frontend/plan/actions/index.js

```diff
@@ -211,7 +211,7 @@ function buildCourseSearchUrl(filterData) {
     const reqs = [];
     if (filterData.selectedReq) {
         for (const key of Object.keys(filterData.selectedReq)) {
-            if (filterData.selectedReq[key] === 1) {
+            if (filterData.selectedReq[key]) { // change from == 1
```

Comment:
No need to add comments with what changed :)
```suggestion
            if (filterData.selectedReq[key]) {
```